packages/koa/README.md

```diff
@@ -0,0 +1,68 @@
+
+## Requirements
+
+  AWS X-Ray SDK Core (aws-xray-sdk-core)
+  Koa 2.x or greater
+
+## AWS X-Ray and Koa
+
+The AWS X-Ray Koa package automatically records information for incoming and outgoing
+requests and responses, via the middleware functions in this package.
+
+The AWS X-Ray SDK Core has two modes - `manual` and `automatic`.
+Automatic mode uses the Continuation Local Storage package (CLS) and automatically
+tracks the current segment and subsegment. This is the default mode.
+Manual mode requires that you pass around the segment reference.
+
+In automatic mode, you can get the current segment/subsegment at any time:
+    var segment = AWSXRay.getSegment();
+
+In manual mode, you can get the base segment off of the request object:
+    var segment = req.segment;
+
+## Sampling rates on routes
+
+Sampling rates are determined by the `aws-xray-sdk-core` package, using the default
+sampling file that is provided, or by overriding this with a custom sampling file.
+For more information on sampling, see aws-xray-sdk-core [README](https://github.com/aws/aws-xray-sdk-node/tree/master/packages/core/README.md).
+
+## Dynamic and fixed naming modes
+
+The SDK requires that a default segment name is set when using middleware.
+If it isn't set, an error is thrown. You can override this value via the `AWS_XRAY_TRACING_NAME`
+environment variable.
+
+    app.use(xrayKoa.openSegment('defaultName'));
+
+The AWS X-Ray SDK Core defaults to a fixed naming mode. This means that each time the middleware creates a new segment for an incoming request,
+the name of that segment is set to the default name. In dynamic mode, the segment name can vary between the host header of the request or the default name.
+For more information about naming modes, see the aws-xray-sdk-core [README](https://github.com/aws/aws-xray-sdk-node/tree/master/packages/core/README.md).
+
+## Automatic mode examples
+
+    var AWSXRay = require('aws-xray-sdk-core');
+    var xrayKoa = require('aws-xray-sdk-koa');
```

Comment:
@willarmiros 
Done.
What would be nice, although a bit of a change is to namepace these libraries under @aws or some other similar namespace so that we don't run into naming collisions with one-offs that people have produced out there.